doc/changelog/07-commands-and-options/11617-toplevel+boot.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  New option ``-boot`` that will not bind the `Coq` library prefix
```

Comment:
I suggest writing explicitly to which tool(s) this new option is added.

Reply:
Done thanks.